clients/drdisas/drdisas.cpp

```diff
@@ -150,15 +201,13 @@ main(int argc, const char *argv[])
             return 1;
         }
     }
-#endif
 
 #if defined(AARCH64) || defined(RISCV64)
     dr_set_vector_length(op_vl.get_value());
 #endif
 
     // XXX i#4021: arm not yet supported.
 #ifdef X86
-    dr_disasm_flags_t syntax = DR_DISASM_DR;
     // Set the syntax if supplied.
     if (!op_syntax.get_value().empty()) {
         if (op_syntax.get_value() == "intel")
```

Comment:
What if the ISA mode is `regdeps`, but the user sets `-syntax intel` or `att` (so, something different than `dr`). Is that still ok? Or should we catch this early and disallow it?

Reply:
Looks like it crashes:
```
$ clients/bin64/drdisas -syntax intel -mode regdeps 00001931 04020204 00000026
Segmentation fault (core dumped)
```

Added an error message and exit if intel or att requested with regdeps.  Will either file an issue or fix the crash separately as it could happen outside of drdisas.